profiler/ann_data.py

```diff
@@ -0,0 +1,23 @@
+from time import perf_counter
+
+import cellxgene_census
+
+import tiledbsoma as soma
+
+census_S3_latest = dict(census_version="latest")
+census_local_copy = dict(uri="/Users/brobatmili/projects/census_data/")
```

Comment:
rm user path

Reply:
since query times can change with census release versions, all profiling comparisons need to be done against a single census data release.  So the data release should be fixed. It should be updated on a reasonable schedule, such as when a new LTS data release is made available.